lib/auth/v2/checkRequestExpiry.js

```diff
@@ -1,24 +1,35 @@
 'use strict'; // eslint-disable-line strict
+const errors = require('../../errors');
 
 function checkRequestExpiry(timestamp, log) {
+    const epochTime = new Date('1970-01-01').getTime();
```

Comment:
We can have this as a global instead, meaning we don't create it every time we go in this function.